app/config/config.yml

```diff
@@ -89,6 +89,7 @@ surfnet_stepup_ra_ra:
 
 mopa_bootstrap:
     form:
+        render_optional_text: false
```

Comment:
"no [combination] of required and optional fields".

Turn off every decoration by default and add it for a specific form when needed.